server/src/reactruntime.js

```diff
@@ -37,6 +37,7 @@ exports.HeadTemplate = class HeadTemplate extends React.Component {
       localeScripts.push(<script key={`l10n-${locale}`} src={this.props.staticLink(`/static/locales/${locale}.js`)} />);
     }
 
+    const wantsAuth = (!this.props.authenticated) || (this.props.authenticated && !this.props.hasFxa && this.props.authFxa);
```

Comment:
nit: could use a newline here.  Having that long line atop of the return looks a little weird.

Reply:
Done